style.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.item {
+  width: 100%;
+  position: relative;
+}
+
+.item2 {
```

Comment:
This is good practice to make sure you have consistency for your button location. 
For future versions of this extension, consider adding control features in the extension dropdown.